components/Nav.vue

```diff
@@ -0,0 +1,186 @@
+<template>
+  <nav>
+    <div class="navbar container mx-auto px-4">
+      <ul id="menu_list">
+        <li><a class="text-lg font-bold" href="#about">Despre Alpha</a></li>
+        <li><a class="text-lg font-bold" href="#testimonials">Testimoniale</a></li>
+        <li><a class="text-lg font-bold" href="#register">Încriere</a></li>
+      </ul>
+      <button id="toggle" @click="onOfMenu(toggle = !toggle)">
+        <span class="nav_btn_actv"></span>
+        <span class="nav_btn_actv"></span>
+      </button>
+      <a href="#home"><img id="logo" src="../static/logo.svg" alt=""></a>
+    </div>
+  </nav>
+</template>
+
+<script>
+export default {
+  data () {
+    return {
+      toggle: false
+    }
+  },
+  methods: {
+    onOfMenu (e) {
+      console.log(this.toggle)
+      if (this.toggle === true) {
+        document.getElementById('menu_list').classList.add('active')
+        document.getElementsByClassName('nav_btn_actv')[0].classList.add('on_off_btn')
+        document.getElementsByClassName('nav_btn_actv')[1].classList.add('on_off_btn')
+      } else {
+        document.getElementById('menu_list').classList.remove('active')
+        document.getElementsByClassName('nav_btn_actv')[0].classList.remove('on_off_btn')
+        document.getElementsByClassName('nav_btn_actv')[1].classList.remove('on_off_btn')
+      }
+    },
+    handleScroll (e) {
+      this.nav = document.querySelector('nav')
+      this.logo = document.getElementById('logo')
+      if (window.scrollY > 0) {
+        this.nav.style.height = '50px'
+        this.logo.style.height = '40px'
+      } else {
+        this.nav.style.height = '100px'
+        this.logo.style.height = '80px'
+      }
```

Comment:
aceeasi resursa de mai sus si pentry style binding.

oricum, nu e deloc indicat ce faci tu aici cu `this.`. Ai putea la fel de bine sa le faci pe toate astea `const`, de exemplu

```
const nav = document.querySelector('nav')
```

dar nu asa se lucreaza cu styles in Vue.